client/index.js

```diff
@@ -72,3 +73,21 @@ if ( appRoot ) {
 		wpBody.insertBefore( noticeContainer, wrap )
 	);
 }
+
+// Set up customer effort score tracking.
+const root = appRoot || embeddedRoot;
+const customerEffortScoreTracksQueue = JSON.parse(
+	window.localStorage.getItem( 'customerEffortScoreTracks' ) || '[]'
+);
```

Comment:
I've seen parsing of local storage wrapped in a try/catch statement in case there is corrupted data. Would be a good idea here.

Reply:
So if `localStorage.getItem()` returns something that isn't null but isn't valid JSON? Seems pretty unlikely and something that should get logged to the console rather than just thrown away.